sos/plugins/ceph.py

```diff
@@ -43,7 +43,7 @@ def setup(self):
             "/var/lib/ceph/",
             "/var/run/ceph/",
             "/etc/calamari/",
-            "/var/log/calamari",
+            "/var/log/calamari/",
```

Comment:
```
# mkdir -p /var/log/calamari
# touch /var/log/calamari/{chtulu,calamari}.log
# sosreport -v --batch --build -e ceph -o ceph
[...]
sosreport build tree is located at : /var/tmp/sosreport-localhost.localdomain-20170314162822

# ll /var/tmp/sosreport-localhost.localdomain-20170314162822/var/log/calamari/
total 0
-rw-r--r--. 1 root root 0 Mar 14 16:27 calamari.log
-rw-r--r--. 1 root root 0 Mar 14 16:27 chtulu.log
```

A more useful change would be to implement `--all-logs` and `--log-size` for the plugin (and get read of the deadcode `logs`option, which also does nothing...).

This would:

* Collect only `calamari.log` and `cthulu.log` with using `--log-size` unless `--all-logs`
* Collect all of `/var/log/calamari` if `--all-logs` is given

These are both implemented by numerous plugins today so there are plenty of examples to take a look at.